runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsOverlay.java

```diff
@@ -188,4 +207,19 @@ else if (plugin.getRoomBlacklist().contains(room.getName().toLowerCase()))
 
 		return panelComponent.render(graphics);
 	}
+
+	private String getRegion(int locationId)
+	{
+		switch (locationId)
+		{
+			case 0:
+				return "US";
+			case 1:
+				return "UK";
+			case 3:
+				return "AUS";
+			default:
```

Comment:
What number is this default case? This seems very unsafe in the event new regions are added.